lesson_9/src/annotation/Runner.java

```diff
@@ -0,0 +1,32 @@
+package annotation;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class Runner {
+    public static void main(String[] args) {
+
+        if(MyService.class.isAnnotationPresent(Version.class)) {
+            MyService service = new MyService();
+
+            try {
+                Field nameField = service.getClass().getDeclaredField("name");
+                Field versionField = service.getClass().getDeclaredField("version");
+                Method method = service.getClass().getDeclaredMethod("thisClassInfo");
+
+                nameField.setAccessible(true);
+                versionField.setAccessible(true);
+                method.setAccessible(true);
+
+                nameField.set(service, "MyServiceName");
+                versionField.set(service, 41259);
```

Comment:
You have version in Version annotation, don't remember the task but was sure you should print it's value